hail/src/main/scala/is/hail/expr/ir/agg/AppendOnlyBTree.scala

```diff
@@ -178,31 +178,27 @@ class AppendOnlyBTree(kb: EmitClassBuilder[_], key: BTreeKey, region: Value[Regi
     val node = get.getCodeParam[Long](1)
     val k = get.getEmitParam(2)
 
-    val cmp = get.newLocal[Int]()
-    val keyV = get.newLocal[Long]()
-
-    def insertOrGetAt(i: Int) = EmitCodeBuilder.scopedVoid(get) { cb =>
-      cb.ifx(isLeaf(node), {
-        cb.assign(keyV, cb.invokeCode(insert, node, const(i), k, const(0L)))
-        cb.assign(cmp, 0)
-      }, {
-        cb.assign(node, loadChild(node, i - 1))
+    get.emitWithBuilder { cb =>
```

Comment:
voidWithBuilder

Reply:
This was an error, `btree_get` returns a value. Fixed now.